solution/ui/prototype/src/views/Resources.vue

```diff
@@ -335,6 +357,7 @@ export default {
     beforeCreate() {},
 
     async created() {
+        console.log("route", this.$route);
```

Comment:
Take it out, leave it not super particular, but probably doesnt need to be in there unless you think we might need it in the future for debugging